README.markdown

```diff
@@ -5573,6 +5573,15 @@ ngx.escape_uri
 
 Escape `str` as a URI component.
 
+Since `v0.10.16rc6`, this function accepts an optional boolean `not_component` argument. When this argument is `true`, these characters bellow will not be escaped.
```

Comment:
I think `no_component` is better?

Reply:
It's still "not_component"?

Reply:
I've thought more about this. Mabye renaming the argument to `not_uri_comp` is a better.